src/searchPage/searchPage.html

```diff
@@ -1,9 +1,8 @@
 <swt-filter-panel></swt-filter-panel>
-<main class="container">
-    <div class="content-wrapper">
-        <swt-searchbar></swt-searchbar>        
-        <h4 class="search-results">Total results</h4>
-        <swt-search-results></swt-search-results>
-    </div>
-</main>
+<div class="content-wrapper">
+    <swt-searchbar></swt-searchbar>
+    <h4 class="search-results">Total results</h4>
```

Comment:
I'd expect search-results to be a class name for search-results directive. Maybe change to search-count to avoid confusion?

Reply:
Updated.